docs/cristin.ttl

```diff
@@ -962,6 +964,77 @@ It is a self reference.""" ;
         entityDescription.reference.doi
     """ .
 
+:antall_sider_totalt
+    a               :cristinfield ;
+    rdfs:comment    "The total number of pages. Possibly null." ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:publiseringsar
```

Comment:
We're still busy tracking down what this is, compared to arstall_online and arstall_trykket, as well as the generic arstall of the result.

Reply:
Should I set it as UNKNOWN then?